core/src/config.jl

```diff
@@ -44,8 +44,8 @@ end
     # [s] Δt for periodic update frequency, including user horizons
     update_timestep::Float64 = 60 * 60 * 24.0
 
-    # optional, default is the path of the TOML
-    toml_dir::String = pwd()
+    # optional, when Config is created from a TOML file, this is its directory
+    relative_dir::String = "."
     input_dir::String = "."
     output_dir::String = "."
```

Comment:
Final nitpick: I think we should use pwd for all three dirs instead of ".". If someone does weird stuff with cwd while running ribasim (for example while coupling) that might otherwise be annoying to debug.

Reply:
Mmmh, actually only relative_dir.
so the behavior from before.

Reply:
Right, I see what you mean: 235a8c2cdad5002917a07192ee4a899c9c430399